Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XlgDebugger/XlgDebuggerState.cs

```diff
@@ -58,7 +60,7 @@ public XlgDebuggerState(DebugLogsAnalyzer analyzer)
         {
             Analyzer = analyzer;
 
-            var logFile = Path.Combine(Path.GetDirectoryName(Analyzer.Input.ExecutionLogPath), "BuildXL.log");
+            var logFile = Path.Combine(Path.GetDirectoryName(Analyzer.Input.ExecutionLogPath), "BuildXL.status.csv");

```

Comment:
>BuildXL.status.csv [](start = 96, length = 18)

I might be missing something, but why what are you doing with this file? `.log` and `.status.csv` are very different files...